msys2/README.md

```diff
@@ -0,0 +1,56 @@
+# Building for Windows using MSYS2
+
+## Installing MSYS2
+Download the installer from [official site](https://www.msys2.org/) or [GitHub Releases page](https://github.com/msys2/msys2-installer/releases).
+
+Run the installer. When done, run "MSYS2 MSYS" from Start menu.  
+
+Update the package database and base packages:
+
+```bash
+pacman -Syu
```

Comment:
@mvov Does msys2 install pacman by default?

Reply:
Yes, pacman is available from the box.